charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -474,34 +519,17 @@ data:
         target_label: pod
       - source_labels: [ __meta_kubernetes_pod_node_name ]
         target_label: node
-      metric_relabel_configs:
-{{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.kubeProxy | indent 6 }}
-
-    - job_name: vpn-connection
```

Comment:
The `vpn-connection` metrics should only be removed when `KonnectivityTunnel: true`
Since there is no vpn when the feature gate is active the following files should not be deployed:
* charts/seed-monitoring/charts/core/charts/prometheus/rules/vpn.rules.yaml
* charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/vpn.rules.test.yaml
* charts/seed-monitoring/charts/grafana/dashboards/operators/vpn-dashboard.json (already done)

for the file
* charts/seed-monitoring/charts/grafana/dashboards/operators/cluster-overview-dashboard.json

there is a graph that shows if the VPN connection is up/down. This can be removed since the VPN dashboard can be used instead.

Reply:
Can you take another look at the `vpn.rules.yaml`?
After reconciling a 1.17 cluster with your PR, I have several firing alerts: `VPNConnectionDown` and `VPNProbeAPIServerProxyFailed`.
I think, what would be even better than just removing the vpn dashboards and rules for cluster with the konnectivity tunnel enabled, would be to make the dashboards and rules more cover both cases, no matter what tunnel implementation is used.
I'm fine with doing this in another PR, but I think, fixing the alerts for VPN cluster before merging would be good.